airbyte-integrations/connector-templates/generator/build.gradle

```diff
@@ -28,7 +28,7 @@ def addScaffoldTemplateTask(name, packageName, outputDirName, scaffoldParams=[])
             }
             exec {
                 workingDir rootDir
-                def cmd = ['./tools/integrations/manage.sh', 'scaffold', name, packageName]
+                def cmd = ['./tools/integrations/manage.sh', 'scaffold', name, packageName, 'uuid']
```

Comment:
This field is expected to be a UUID and not the string `uuid`. Does it matter here? If anything can be put here, should we be more descriptive in the string that this can be any string like `any string with uuid format` or whatever?

Reply:
What matters is to have an artificial static value here to have idempotent results when a scaffolds happen in the CI, as the CI compare the scaffolds output to the previously versionned scaffolds. I'll change it to a more intuitive value as you suggested.